tests/integration.sh

```diff
@@ -949,13 +949,16 @@ function register_receiver() {
 function test_send() {
     set -e
     local contract_addr="$1"
+    local skip_register_receiver="$2"
 
     log_test_header
 
-    local receiver_addr
-    receiver_addr="$(create_receiver_contract)"
-    # receiver_addr='secret17k8qt6aqd7eee3fawmtvy4vu6teqx8d7mdm49x'
-    register_receiver "$receiver_addr" "$contract_addr"
+    local receiver_addr="$(create_receiver_contract)"
+    local receiver_hash="$(secretcli q compute contract-hash $receiver_addr | sed 's/^0x//')"
```

Comment:
The reason i separated the `local name` line from `name="$(command)"` line, is because `local` overrides the return code of `command`, so even if it fails, the script will not crash. Please make sure you declare local variables separately from their assignment, when assigning the output of commands.

Reply:
Done